pkgs/applications/misc/rxvt_unicode/wrapper.nix

```diff
@@ -12,8 +12,10 @@ in symlinkJoin {
 
   postBuild = ''
     wrapProgram $out/bin/urxvt \
+      --set PERL5LIB "${perlPackages.makePerlPath perlDeps}" \
```

Comment:
Have you considered augmenting rather than overriding the search path, to support picking up deps not managed by Nix? Or is that undesirable?

Reply:
No, and it would probably be better to do it but I don't know how.

Reply:
Something like `--prefix PERL5LIB : ${makePerlPath perlDeps}`.

The various flags understood by `makeWrapper`/`wrapProgram` are documented at https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/setup-hooks/make-wrapper.sh#L27.

Reply:
Thank you. I did it.